docs-content/getting-started/fullstack-frameworks/next-js.md

```diff
@@ -274,6 +274,12 @@ export default function RootLayout({ children }: { children: React.ReactNode })
 }
 ```
 
+## Configure `inlineImages`
+
+We use a package called [rrweb](https://www.rrweb.io/) to record web sessions. rrweb enables inlining images into session replay to improve performance; however, the inlined images can cause CORS issues in some situations.
+
+We currently default `inlineImages` to `true`. Explicitly set `inlineImages={false}` if you run into trouble loading images on your page while Highlight is running. This will degrade tracking on localhost and other domains that are inaccessible to `app.highlight.io`.
```

Comment:
> We currently default `inlineImages` to `true`.

I don't think this is right; we only default to `true` when you are on `localhost` to be able to replay the session over the network or after your server is stopped.